Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs

```diff
@@ -69,7 +74,13 @@ public OSMBuildingData(List<Vector3> footprint, Vector2[][] holes, OSMTags tags)
         this.grammar = Grammars.relations;
         MakeRelative();
         SetHeightAndLevels(tags.height, tags.levels);
+        tags.levels -= 2;
         SetElevation(footprint);
+        
+        if (tags.building == "museum" || tags.tourism == "museum")
+        {
+	        this.grammar = Grammars.museum;
```

Comment:
Not sure this is DRY as its a repeat of the clumsy inelegant solution from previous comment